recipes/cwl-utils/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cwl-utils" %}
+{% set version = "0.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cwl-utils-{{ version }}.tar.gz
+  sha256: 43e57318db207310e46726df1081c55ceddebab888e09484d1175f4c9a103384
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
List console script entry points in the build section.

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Hmm, I couldn't find an example in the link above @carterbox. Had a look at the recently updated feedstock projects, but couldn't find an example of how to use the console script entry points in the build section. Would you have an example to share, please? Thanks.

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Reply:
Thanks @carterbox! I've updated the recipe, and also bumped to 0.19 (checked the requirements too)